src/main/java/mekanism/common/item/ItemModule.java

```diff
@@ -49,15 +50,16 @@ public Rarity getRarity(@Nonnull ItemStack stack) {
     @Override
     public void appendHoverText(@Nonnull ItemStack stack, Level world, @Nonnull List<Component> tooltip, @Nonnull TooltipFlag flag) {
         if (MekKeyHandler.isKeyPressed(MekanismKeyHandler.detailsKey)) {
-            tooltip.add(new TextComponent("Supported by:"));
-            for (Item item : MekanismAPI.getModuleHelper().getSupported(getModuleData())) {
-                tooltip.add(item.getName(new ItemStack(item)));
+            tooltip.add(MekanismLang.MODULE_SUPPORTED.translateColored(EnumColor.BRIGHT_GREEN));
+            IModuleHelper moduleHelper = MekanismAPI.getModuleHelper();
+            for (Item item : moduleHelper.getSupported(getModuleData())) {
+                tooltip.add(MekanismLang.GENERIC_LIST.translate(item.getName(new ItemStack(item))));
             }
-            Set<ModuleData<?>> conflicting = MekanismAPI.getModuleHelper().getConflicting(getModuleData());
-            if(!conflicting.isEmpty()){
-                tooltip.add(new TextComponent("Conflicting:"));
+            Set<ModuleData<?>> conflicting = moduleHelper.getConflicting(getModuleData());
+            if (!conflicting.isEmpty()) {
+                tooltip.add(MekanismLang.MODULE_CONFLICTING.translateColored(EnumColor.RED));
                 for (ModuleData<?> module : conflicting) {
-                    tooltip.add(module.getTextComponent());
+                    tooltip.add(MekanismLang.GENERIC_LIST.translate(module.getTextComponent()));
```

Comment:
Explicit `getTextComponent` call shouldn't be needed anymore

Reply:
lol, even thought of that when I was renaming the GENERIC_LIST just forgot when putting it in